update-versions.py

```diff
@@ -103,16 +104,27 @@ def nix_prefetch(args):
 g = github.Github(auth=auth)
 repo = g.get_repo("hashicorp/terraform")
 # TODO: Drop "v" prefix first
-# TODO: Combine the output of the following endpoints to get a list of active
-# versions
-# https://api.github.com/repos/hashicorp/terraform/tags
-# https://endoflife.date/api/terraform.json
```

Comment:
Not sure if I captured the whole idea presented here. In any case, using the tags endpoint could be problematic since we wouldn't be able to filter out draft tags.